hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/keyvalue/helpers/TestChunkUtils.java

```diff
@@ -124,6 +126,8 @@ void concurrentReadOfSameFile() throws Exception {
       executor.shutdownNow();
     }
     assertFalse(failed.get());
+
+    ChunkUtils.clearStripedLock();
```

Comment:
Instead of clearing, add a check to see if the read and locks have been released.

Reply:
Also, to test isolation it makes sense to have multiple writers writing different data at incrementing offset while a single reader thread checks if it can read partial writes or corrupted writes.

Reply:
@kerneltime Thanks for the reivew, I added a new UT：org.apache.hadoop.ozone.container.keyvalue.helpers.TestChunkUtils#concurrentReadWriteOfSameFile